gatsby-config.js

```diff
@@ -8,82 +8,79 @@ require("dotenv").config({
 })
 
 module.exports = {
-    pathPrefix: "/openousd-site",
-    siteMetadata: {
-        title: `OpenOUSD`,
-        description: `OpenOUSD aims to bring greater transparency to the Oakland Unified School District's central office.`,
-        author: `@OpenOUSD`,
-        latestSchoolYear: `2019-20`
+  pathPrefix: "/openousd-site",
+  siteMetadata: {
+    title: `OpenOUSD`,
+    description: `OpenOUSD aims to bring greater transparency to the Oakland Unified School District's central office.`,
+    author: `@OpenOUSD`,
+    latestSchoolYear: `2019-20`,
+  },
+  plugins: [
+    {
+      resolve: `gatsby-plugin-google-analytics`,
+      options: {
+        trackingId: "UA-69150718-3",
+        // Defines where to place the tracking script - `true` in the head and `false` in the body
+        head: false,
+      },
     },
-    plugins: [
-        {
-            resolve: `gatsby-plugin-google-analytics`,
-            options: {
-                trackingId: "UA-69150718-3",
-                // Defines where to place the tracking script - `true` in the head and `false` in the body
-                head: false,
-            },
-        },
-        {
-          resolve: 'gatsby-plugin-sass',
-          options: {
-            data: `@import "./src/styles/_variables.scss";`,
-          }
-        },
-        `gatsby-plugin-react-helmet`,
-        {
-            resolve: `gatsby-source-filesystem`,
-            options: {
-                name: `images`,
-                path: `${__dirname}/src/images`,
-            },
-        },
-        {
-            resolve: "gatsby-source-filesystem",
-            options: {
-                path: `${__dirname}/data/`,
-            },
-        },
-        `gatsby-transformer-sharp`,
-        `gatsby-plugin-sharp`,
-        {
-            resolve: `gatsby-plugin-intl`,
-            options: {
-                path: `${__dirname}/src/translations`,
```

Comment:
Oops, prettier reformatted the file on save. This is the only line that changed. `/src/translations` > `/src/languages`